BankAccount/Account.cs

```diff
@@ -46,7 +46,7 @@ public decimal GetCurrentBalance()
         /// <summary>
         /// Represents the transactions held in the bank account, read-only status stops list being modified outside of this class.
         /// </summary>
-        public IReadOnlyCollection<ITransaction> Transactions { get { return _Transactions.AsReadOnly(); } }
+        public IReadOnlyCollection<ITransaction> Transactions { get { return _Transactions; } }
```

Comment:
I had to look this one up to make a determination. I believe AsReadOnly() should stay after reading this:
https://stackoverflow.com/questions/17398428/listt-asreadonly-vs-ireadonlycollectiont

Reply:
I read up also... But couldn't find a definitive answer... Merged before I read your reply